examples/getstarted/src/plugins/mycustomfields/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "mycustomfields",
+  "version": "0.0.0",
+  "description": "This is the description of the plugin.",
+  "strapi": {
+    "name": "mycustomfields",
+    "description": "Description of Mycustomfields plugin",
+    "kind": "plugin",
+    "displayName": "Mycustomfields"
```

Comment:
Would be nice to have more custom description & display name here. When we'll merge on master it will let other devs know where this plugin comes from